### DISCUSSION TITLE
Compensate Bluetooth delay

### DISCUSSION THREAD

**Discussion by Pijotr** [rern/rAudio#708] [General]
1st: Hello, I would like to go back to rune, but there is one feature wich is necessary for me. Streaming audio from videofiles to the Raspberry. With moodeaudio it is very easy to compensate the delay.
In rune is it not possible. Has anybody an idea how to fix ist?
Sound is much better than in moode but I need this feature.

2nd: Firestick did not finde the rune bluetooth. Any idea?

Thanks

**Comment by rern** [comment]
1st - Which one, AirPlay, Bluetooth or UPnP?
2nd - System > Bluetooth - enable
  - When `bluetoothctl` shows under the label, click it to show status
  - If no status, try reboot.
